sql/updates/latest-dev.sql

```diff
@@ -16,3 +16,65 @@ DROP FUNCTION IF EXISTS _timescaledb_functions.invalidation_process_cagg_log(int
 DROP FUNCTION IF EXISTS _timescaledb_functions.invalidation_process_hypertable_log(integer,integer,regtype,integer[],bigint[],bigint[]);
 DROP FUNCTION IF EXISTS _timescaledb_functions.invalidation_process_hypertable_log(integer,integer,regtype,integer[],bigint[],bigint[],text[]);
 DROP FUNCTION IF EXISTS _timescaledb_functions.hypertable_invalidation_log_delete(integer);
+
+--
+-- Rebuild the catalog table `_timescaledb_catalog.continuous_aggs_bucket_function`
+--
+CREATE TABLE _timescaledb_catalog._tmp_continuous_aggs_bucket_function AS
+    SELECT
+      mat_hypertable_id,
+      CASE WHEN experimental THEN 'timescaledb_experimental'::name ELSE '_timescaledb_functions'::name END,
+      name,
+      bucket_width,
+      origin,
+      NULL::text AS bucket_offset,
+      timezone,
+      false::bool AS bucket_fixed_width,
```

Comment:
`false` is `boolean` already, probably doesn't need a cast.

Reply:
Thanks, changed.